### ISSUE TITLE
Request to have New-BcContainer check if image is available before deleting existing container

### ISSUE THREAD

**Issue by geschwint** [microsoft/navcontainerhelper#2108]
**Describe the issue**
New-BcContainer deletes the existing container even if no image is available

**Scripts used to create container and cause the issue**
```
New-BcContainer `
    -accept_eula `
    -imagename...
```
Happens in line 509: Remove-BcContainer $containerName
Should maybe be optional (Parameter), like give error if exist or at least not remove unless image is available. Not a major but with BC19.0 I was to early today so the container was dropped but did not get a new one.

**Full output of scripts**
```
N/A...
```

**Screenshots**
N/A

**Additional context**
N/A

**Comment by freddydk** [comment]
is this because you are using your own build images?

**Comment by geschwint** [comment]
nop. I had a container setup based on insider builds for 19 with data and all. So when I got the message this morning about the general availability of 19, I'd make an update. So more out of convenience, have the New-Bc... first check if image is available before delete existing container to save the inconvenience of having to recreate a bcinsider container and load data again. Like an option...

**Comment by freddydk** [comment]
We do not use images anymore - we use artifacts and in New-BcContainer, you give it a set of artifacts and it will create the container based on that.
If artifacts doesn't exist - it is discovered early (get-bcartifacturl returns empty string) and you are right, that the old container is removed before reporting that "You have to specify artifactUrl or imageName when creating a new container."
I will move that test up.

**Comment by freddydk** [comment]
Shipped in 2.0.17.preview
